src/explore-education-statistics-common/src/components/form/__tests__/FormRadioSearchGroup.test.tsx

```diff
@@ -76,15 +74,15 @@ describe('FormRadioSearchGroup', () => {
 
     const searchInput = screen.getByLabelText('Search options');
 
-    await userEvent.type(searchInput, '[[');
+    await user.type(searchInput, '[[');
 
     await waitFor(() => {
       expect(screen.queryAllByLabelText(/Test radio/)).toHaveLength(0);
     });
   });
 
   test('providing a search term does not remove radio buttons that have already been checked', async () => {
-    render(
+    const { user } = render(
```

Comment:
ah cool, didn't realise you can do this with jest. Assume this and `@testing-library/user-event` are identical?

Reply:
Yeah, this is just a quick way to do `userEvent.setup()`, which is the recommended way to use user events now. See https://github.com/dfe-analytical-services/explore-education-statistics/blob/dev/src/explore-education-statistics-common/test/render.tsx